contracts/lbp/WrapperFactory.sol

```diff
@@ -0,0 +1,119 @@
+/*
+
+██████╗░██████╗░██╗███╗░░░███╗███████╗██████╗░░█████╗░░█████╗░
+██╔══██╗██╔══██╗██║████╗░████║██╔════╝██╔══██╗██╔══██╗██╔══██╗
+██████╔╝██████╔╝██║██╔████╔██║█████╗░░██║░░██║███████║██║░░██║
+██╔═══╝░██╔══██╗██║██║╚██╔╝██║██╔══╝░░██║░░██║██╔══██║██║░░██║
+██║░░░░░██║░░██║██║██║░╚═╝░██║███████╗██████╔╝██║░░██║╚█████╔╝
+╚═╝░░░░░╚═╝░░╚═╝╚═╝╚═╝░░░░░╚═╝╚══════╝╚═════╝░╚═╝░░╚═╝░╚════╝░
+
+*/
+
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+// solium-disable linebreak-style
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "../utils/CloneFactory.sol";
+import "openzeppelin-solidity/contracts/access/Ownable.sol";
+import "./LBPWrapper.sol";
+import "hardhat/console.sol";
+
+
+contract WrapperFactory is CloneFactory, Ownable {
+
+    address public wrapperMasterCopy;
+
+    address public LBPFactory;
+    uint256 public swapFeePercentage;
+    bool public isInitialized;
```

Comment:
What do we need this state variable for? It seems to me as if the `WrapperFactory` is always initiliazed as soon as it has been deployed (due to the constructor). Can `isInitialized` ever be set to `false`?

Reply:
Not yet, but kept it for some other function which is now removed. Will check it out.

Reply:
@codebuster22 Is it okay to remove this?

Reply:
Removed the variable